src/tests/token/erc20.cairo

```diff
@@ -1,4 +1,5 @@
 pub(crate) mod common;
 
+mod test_dual20;
```

Comment:
```suggestion
// mod test_dual20;
```

Let's comment out this module for now, since test utils won't be available and the file will fail to compile

Reply:
Not sure what you mean. The file does compile.